backends/self/self_backend.cpp

```diff
@@ -16,7 +16,7 @@ class Send {
     void *libbuf;
     int count;
     MPI_Datatype datatype;
-    int tag; // TODO: handle tag, just change map to <pair<tag, comm>, vector<Send*>>
```

Comment:
The self backend doesn't seem to be updated to handle tag, so we should keep this comment